java/plasma/pom.xml

```diff
@@ -18,6 +18,7 @@
     </parent>
     <artifactId>arrow-plasma</artifactId>
     <name>Arrow Plasma Client</name>
+    <description>Java client for the Plasma object store.</description>
```

Comment:
I think this is experimental/contrib. It is still written completing ignoring other parts of arrow.

Reply:
Revised.